.github/workflows/main.yaml

```diff
@@ -36,8 +34,6 @@ jobs:
         with:
           node-version: ${{ matrix.node }}
           cache: npm
-      - name: Install npm latest
-        run: npm i -g npm
```

Comment:
That is needed during CI because it requires npm 9. Often older Node releases still ship with an older npm version by default.

Reply:
I pinned it instead, as suggested in https://github.com/terser/html-minifier-terser/pull/171#discussion_r1382474078

Reply:
I'm not sure if we should pin npm here. I would prefer to not pin its version.

What do you think @sibiraj-s?